manager/assets/modext/widgets/security/modx.panel.access.policy.template.js

```diff
@@ -41,58 +37,58 @@ MODx.panel.AccessPolicyTemplate = function(config) {
                     html: '<p>'+_('policy_template.desc')+'</p>'
                     ,xtype: 'modx-description'
                 },{
-					xtype: 'panel'
-					,border: false
-					,cls:'main-wrapper'
-					,layout: 'form'
-					,defaults:{ anchor: '100%' }
-					,labelAlign: 'top'
-					,labelSeparator: ''
-					,items: [{
-						xtype: 'hidden'
-						,name: 'id'
-					},{
-						xtype: 'textfield'
-						,fieldLabel: _('name')
-						,description: MODx.expandHelp ? '' : _('policy_template_desc_name')
-						,name: 'name'
-						,id: 'modx-policy-template-name'
-						,maxLength: 255
-						,enableKeyEvents: true
-						,allowBlank: false
-						,listeners: {
-							'keyup': {scope:this,fn:function(f,e) {
-								Ext.getCmp('modx-policy-template-header').getEl().update(_('policy')+': '+f.getValue());
-							}}
-						}
-					},{
+                    xtype: 'panel'
+                    ,border: false
+                    ,cls:'main-wrapper'
+                    ,layout: 'form'
+                    ,defaults:{ anchor: '100%' }
+                    ,labelAlign: 'top'
+                    ,labelSeparator: ''
+                    ,items: [{
+                        xtype: 'hidden'
+                        ,name: 'id'
+                    },{
+                        xtype: 'textfield'
+                        ,fieldLabel: _('name')
+                        ,description: MODx.expandHelp ? '' : _('policy_template_desc_name')
+                        ,name: 'name'
+                        ,id: 'modx-policy-template-name'
+                        ,maxLength: 255
+                        ,enableKeyEvents: true
+                        ,allowBlank: false
+                        ,listeners: {
+                            'keyup': {scope:this,fn:function(f,e) {
+                                Ext.getCmp('modx-policy-template-header').getEl().update(Ext.util.Format.htmlEncode(f.getValue()));
```

Comment:
This is incorrect, should be: `Ext.getCmp("modx-header-breadcrumbs").updateHeader(Ext.util.Format.htmlEncode(f.getValue()));`